sys/net/if_media.h

```diff
@@ -442,6 +443,7 @@ struct ifmedia_description {
 	{ IFM_1000_SX,	"1000baseSX" },					\
 	{ IFM_1000_LX,	"1000baseLX" },					\
 	{ IFM_1000_CX,	"1000baseCX" },					\
+	{ IFM_1000_BX,	"1000baseBX" },					\
```

Comment:
I agree with the change to BX, for instance LX can come in 2km or extended distances as well.  By convention it seems we use the trailing numbers for lanes.